From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: enthropy7 <enthropy7@users.noreply.github.com>
Date: Wed, 26 Aug 2026 13:50:38 +0300
Subject: dt-bindings: display: allwinner: add H3/H5 TV encoder and H3

 mixer1

The H3 and H5 carry the same TV encoder the A10 binding already
describes, and the H3 has a second DE2 mixer alongside the documented
mixer0. Add the three compatibles so the device trees that use them
validate.

Signed-off-by: enthropy7 <enthropy7@users.noreply.github.com>
---
 Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tv-encoder.yaml | 7 +++++--
 Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-de2-mixer.yaml | 1 +
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tv-encoder.yaml b/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tv-encoder.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tv-encoder.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tv-encoder.yaml
@@ -4,7 +4,7 @@
 $id: http://devicetree.org/schemas/display/allwinner,sun4i-a10-tv-encoder.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Allwinner A10 TV Encoder
+title: Allwinner A10/H3/H5 TV Encoder
 
 maintainers:
   - Chen-Yu Tsai <wens@csie.org>
@@ -12,7 +12,10 @@ maintainers:
 
 properties:
   compatible:
-    const: allwinner,sun4i-a10-tv-encoder
+    enum:
+      - allwinner,sun4i-a10-tv-encoder
+      - allwinner,sun8i-h3-tv-encoder
+      - allwinner,sun50i-h5-tv-encoder
 
   reg:
     maxItems: 1
diff --git a/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-de2-mixer.yaml b/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-de2-mixer.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-de2-mixer.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-de2-mixer.yaml
@@ -16,6 +16,7 @@ properties:
       - allwinner,sun8i-a83t-de2-mixer-0
       - allwinner,sun8i-a83t-de2-mixer-1
       - allwinner,sun8i-h3-de2-mixer-0
+      - allwinner,sun8i-h3-de2-mixer-1
       - allwinner,sun8i-r40-de2-mixer-0
       - allwinner,sun8i-r40-de2-mixer-1
       - allwinner,sun8i-v3s-de2-mixer
-- 
Armbian

